pose_prediction/plugin.cpp

```diff
@@ -33,6 +39,11 @@ class pose_prediction_impl : public pose_prediction {
         }
     }
 
+    ~pose_prediction_impl() {
+        // pose_output_post.close();
+        // pose_output_post_offset.close();
+    }
+
```

Comment:
```suggestion
```
Remove empty constructor if not needed.
In particular, I believe those files will be closed in the default destructor.